pkgs/top-level/all-packages.nix

```diff
@@ -13660,6 +13660,50 @@ with pkgs;
     llvmPackages = llvmPackages_14;
   };
 
+  clang-tools-python = callPackage ../development/tools/clang-tools/python.nix {
+    llvmPackages = llvmPackages_latest;
+  };
+
+  clang-tools-python_5 = callPackage ../development/tools/clang-tools/python.nix {
+    llvmPackages = llvmPackages_5;
+  };
+
+  clang-tools-python_6 = callPackage ../development/tools/clang-tools/python.nix {
+    llvmPackages = llvmPackages_6;
+  };
+
+  clang-tools-python_7 = callPackage ../development/tools/clang-tools/python.nix {
+    llvmPackages = llvmPackages_7;
+  };
+
+  clang-tools-python_8 = callPackage ../development/tools/clang-tools/python.nix {
+    llvmPackages = llvmPackages_8;
```

Comment:
Why don't we move it under llvmPackages?

Reply:
Sounds great.

Not sure what would be the proper way to move it into `llvmPackages`. It's quite confusing to see such one-version-per-folder form of packaging.

Reply:
Just moved `clang-tools` and `clang-tools-python` into `llvmPackages`.

Reply:
Update: Strange errors occurred.

Reply:
Addressed.